style.css

```diff
@@ -0,0 +1,100 @@
+html{
+    height: 100%;
+}
+
+body{
+    padding-top: 4%;
+    color: white;
+    background-image:linear-gradient(
+        90deg,
+        rgba(58, 58, 158, 0.8),
+        rgba(136, 136, 206, 0.7)
+    ),
+    url(/tribute-page-main-image.jpg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-attachment: fixed;
+}
+
+h1, p {
```

Comment:
[ ] The comment should be phrased politely, as suggestions instead of orders. `For example:"I think it would be better to use semantic tags"`.kindly fix that 🤓.
[ ] always include an explanation, a "why", for each requested change. `For example: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible. `kindly fix that🤓